addons/l10n_lb/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "Lebanon - Accounting",
+    "countries": ["lb"],
+    "version": "1.0",
+    "description": """
+This is the base module to manage the accounting chart for Lebanon in Odoo.
+==============================================================================
+Lebanon accounting basic charts,taxes and localization.
+Activates:
+ * Chart of Accounts
+ * Taxes
+ * Fiscal Positions
+    """,
+    "category": "Accounting/Localizations/Account Charts",
+    "depends": ["account"],
+    "data": ["data/res.country.state.csv"],
+    "demo": ["demo/demo_company.xml"],
```

Comment:
I would appreciate if you leave space for future additions without breaking the blame, like this
```suggestion
    "data": [
        "data/res.country.state.csv",
    ],
    "demo": [
        "demo/demo_company.xml",
    ],
```